packages/sitecore-jss/src/i18n/graphql-dictionary-service.test.ts

```diff
@@ -65,24 +65,22 @@ describe('GraphQLDictionaryService', () => {
     });
   });
 
-  // TODO: there is a known issue with mcache
   it('should use cache', async () => {
     nock(endpoint)
       .post('/', /GetSiteRoot/gi)
       .reply(200, siteRootQueryResponse);
 
     nock(endpoint)
       .post('/', /DictionarySearch/gi)
-      .times(2)
       .reply(200, dictionaryQueryResponse);
 
-    const service = new GraphQLDictionaryService({ endpoint, siteName });
+    const service = new GraphQLDictionaryService({ endpoint, siteName, cacheTimeout: 2 });
```

Comment:
Might be good to explicitly set `cacheEnabled: true` here to match test case "should use cache" (similar to rest-dictionary-service tests)

Reply:
I added specified this property